src/definitions.ts

```diff
@@ -106,6 +107,18 @@ export interface RadarUser {
   postalCode?: RadarRegion;
 }
 
+export interface RadarTrip {
+  _id: string;
+  externalId: string;
+  metadata?: object;
+  destinationGeofenceTag?: string;
+  destinationGeofenceExternalId?: string;
+  mode?: string;
+  etaDistance?: number;
+  etaDuration?: number;
```

Comment:
Is there a reason this isn't the same as the server response?
```js
eta: {
  duration: number;
  distance: number;
}
```

Reply:
good catch

Reply:
Yeah, here's how it's serialized right now: https://github.com/radarlabs/radar-sdk-ios/blob/master/RadarSDK/RadarTrip.m#L178 https://github.com/radarlabs/radar-sdk-android/blob/master/sdk/src/main/java/io/radar/sdk/model/RadarTrip.kt#L163